vitest.config.js

```diff
@@ -0,0 +1,11 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    environment: 'jsdom',
```

Comment:
Tests need a bit of bashing to work, you should probably look into the different test environments and pick the one that most matches your use case.